### PR TITLE
Typecheck negative cosine similarity

### PR DESCRIPTION
**Pull request by philippmwirth**
# Typecheck negative cosine similarity

### CODE REVIEW

File: tests/loss/test_negative_cosine_similarity.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
+import pytest
 import torch
 
 from lightly.loss import NegativeCosineSimilarity
 
 
-class TestNegativeCosineSimilarity(unittest.TestCase):
-    def test_forward_pass(self):
+class TestNegativeCosineSimilarity:
+    def test_forward_pass(self) -> None:
```
Comment (MalteEbner) [inline tests/loss/test_negative_cosine_similarity.py:8]: Again: best parametric the unittests.

**Review by MalteEbner — COMMENTED** [review]
Again: best parametric the unittests.

**Review by MalteEbner — APPROVED** [review]
